components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/listener/IdentityOathEventListener.java

```diff
@@ -140,9 +130,7 @@ public boolean doPostSetUserClaimValues(String userName, Map<String, String> cla
         if (!isEnable()) {
             return true;
         }
-        return revokeTokensOfLockedUser(userName, userStoreManager) &&
-                revokeTokensOfDisabledUser(userName, userStoreManager)
-                && removeUserClaimsFromCache(userName, userStoreManager);
+        return OAuthUtil.removeUserClaimsFromCache(userName, userStoreManager);
```

Comment:
Isn't this happening in IdentityOauthEventHandler?

Reply:
fixed it. fixed ad removeUserclaim happen only in IdentityOauthEventHandler